app/page/info/activity.tsx

```diff
@@ -1,5 +1,6 @@
 import { useAccount, useUserOperationLogs } from "~app/storage"
 import { UserOperation } from "~packages/bundler"
+import { getChainName } from "~packages/network/util/chain"
```

Comment:
nit: I think it is no need to slice out sub-package at this moment.

```suggestion
import { getChainName } from "~packages/network/util"
```